docs_app/content/tutorial/4-what-is-an-operator.md

```diff
@@ -0,0 +1,71 @@
+# 4. What Is An Operator?
+
+## TLDR:
+
+- Observables are a container for a collection of values over time.
+- Operators are functions that take an [`Observable`](API) and return a new [`Observable`]. `(source: Observable<A>) => Observable<B>`.
+- Operators are applied through the [`pipe` method](API).
```

Comment:
Consider adding one more here that describes the *purpose* of operators before discussing the semantics of operators. Something like: "Operators transform each item emitted from an Observable."